lorentz_force/lorentz_unit_tests.py

```diff
@@ -0,0 +1,49 @@
+"""
```

Comment:
Consider adding tests for edge cases or more general situations such as:
When charge is zero.
When velocity is zero.
When magnetic field is zero.
Negative charges